storage_media/views/storage_media_view.xml

```diff
@@ -23,18 +23,8 @@
                     />
                 </header>
                 <group>
-                    <field name="id" invisible="True" />
-                    <field
-                        name="data"
-                        filename="name"
-                        attrs="{'invisible': [('id', '!=', False)]}"
```

Comment:
why remove this?

Reply:
I will check this issue again

Reply:
ok so you caught a bug, update the description about that then